src/sgx/rust_sgx_bridge/rust_sgx_bridge.cpp

```diff
@@ -0,0 +1,200 @@
+#include <stdio.h>
```

Comment:
Copyright header missing. Fix general spacing and indentation issues in code. for example "if(x ==0) {" should be "if (x == 0) {". Follow google code style.